dot/core/service.go

```diff
@@ -342,6 +344,7 @@ func (s *Service) handleChainReorg(prev, curr common.Hash) error {
 			decExt := &ctypes.Extrinsic{}
 			decoder := cscale.NewDecoder(bytes.NewReader(ext))
 			if err = decoder.Decode(&decExt); err != nil {
+				logger.Error(err.Error())
```

Comment:
this seems to be a returnable error for me.  How do we have a transaction in here that doesn't decode?